src/patternfly/components/VerticalNav/docs/code.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+
+The vertical navigation list contains list items, links, icons and text. Icons, along with text, must be included for the list items to align properly. Do not omit these elements. The chevron is required for list items containing subnavigation. 
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
```

Comment:
Do we need role navigation to the `nav` element? is it necessary in modern browsers?

Reply:
Its use is suggested according to https://a11yproject.com/posts/aria-landmark-roles/